tests/components/mqtt/test_mixins.py

```diff
@@ -335,6 +335,9 @@ async def test_default_entity_and_device_name(
 
     # Assert that no issues ware registered
     assert len(events) == 0
+    await hass.async_block_till_done()
+    # Assert that no issues ware registered
+    assert len(events) == 0
```

Comment:
Is this related?

Reply:
Yes, if we don't wait here, the test will fail because it now running the task in the background